examples/standalone-rhel-aurora/provider.tf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+
+provider "aws" {
+  assume_role {
+    role_arn = var.aws_role_arn
+  }
+
+  default_tags {
+    tags = local.common_tags
+  }
+}
```

Comment:
🚧 We should not create a provider here. After I deployed the Aurora example with this branch, and wanted to tear it down again, I am facing the following issue: 

```
Error: Provider configuration not present
To work with module.tfe.module.standalone_rhel_aurora.module.aurora_database[0].aws_security_group_rule.postgresql_ingress (orphan) its original provider configuration at module.tfe.provider["registry.terraform.io/hashicorp/aws"] is required, but it has been removed. This occurs when a provider configuration is removed while objects created by that provider still exist in the state. Re-add the provider configuration to destroy module.tfe.module.standalone_rhel_aurora.module.aurora_database[0].aws_security_group_rule.postgresql_ingress (orphan), after which you can remove the provider configuration again.
```

If I understand correctly, we should require a provider instead of instantiating one. Please see how the other examples handle this, such as [standalone-airgap-dev](https://github.com/hashicorp/terraform-aws-terraform-enterprise/blob/main/examples/standalone-airgap-dev/versions.tf). 

Also, when I consult the [terraform docs](https://developer.hashicorp.com/terraform/language/modules/develop/providers), I find: 

> A module intended to be called by one or more other modules must not contain any provider blocks.

Please adapt this to follow the other examples.